test/compare-numbers.test.js

```diff
@@ -0,0 +1,44 @@
+// IMPORT MODULES under test here:
+// import example from '../src/example.js';
+import compareNumbers from './compare-numbers.js';
+
+const test = QUnit.test;
+
+test('Should return true if guess is correct', function(assert) {
```

Comment:
Not return true or false, but either 0, 1, or -1. Super minor, but it's better to practice good test writing early on before you have dozens of tests!